teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+class Teacher < Person
+  def initialize(name, age, specialization)
+    super(name, age)
```

Comment:
- [ ] The `name` parameter should be optional always plus it also makes sense to inquire of the student's name also, so that means it should be after the compulsory ones (age and specialization).

  ```
  def initialize(specialization, age, name = 'Unknown')
    super(age, name)
  end
  ```